### PR TITLE
feat: document plugin type badges

### PR DESCRIPTION
**Pull request by Skaiir**
Context:

image

### CODE REVIEW

**Review by MaxTru — CHANGES_REQUESTED** [review]
Would it not be cooler to actually see all the badges? Perhaps a table or so?

**Comment by nikku** [comment]
> Would it not be cooler to actually see all the badges? Perhaps a table or so?

Yes, including MARDOWN markup if possible. Makes it even easier to adopt hem.

File: README.md [hunk 1]
```diff
@@ -39,10 +39,29 @@ npm run all
 
 ## Before you Publish
 
-* [ ] Clearly state which Camunda Modeler version your plug-in is compatible with and which type of plugin it is by changing the badges at the top of this README 
+* [ ] Clearly state which Camunda Modeler version your plug-in is compatible with and which type of plugin it is by changing the badges at the top of this README
+  * Mark your plugin using a type badge, so to show your users, what Camunda Modeler features it extends. You can find a best-practice list of typical badges [below](#badges-to-indicate-typical-plugin-types).
+  * If your plugin is `BPMN`/`DMN`/`React` but also includes menu entries or styles, just use `BPMN`/`DMN`/`React` in the badge.
```
Comment (MaxTru) [inline README.md]: Let's simplify

```suggestion
* [ ] Clearly state which Camunda Modeler version your plug-in is compatible with and which type of plugin it is by changing the badges at the top of this README. See [best-practice examples below](#badges-to-indicate-typical-plugin-types).
```

**Review by MaxTru — CHANGES_REQUESTED** [review]

**Review by MaxTru — CHANGES_REQUESTED** [review]
Can we please consistently use `plugin` instead of `plug-in`. Currently it seems super random which term we use, but `plugin` seems to be the leading one

**Review by MaxTru — APPROVED** [review]

**Pull request merged**